tests/test_vistas.py

```
@@ -0,0 +1,47 @@
+    "/login",
+    "/logout",
+    "/dashboard",
+    "/requisiciones",
+    "/requisiciones/historial",
+    "/requisiciones/pendientes_cotizar",
+    "/requisiciones/cotizadas",
+    "/admin/usuarios",
+])
+def test_vistas_basicas_cargan(client, ruta):
+    # Crear y loguear como superadmin para todas las vistas
+    user = crear_usuario(client, "superadmin1", "Superadmin")
```

Ruff (0.11.9)

Local variable `user` is assigned to but never used

Remove assignment to unused variable `user`

(F841)